src/expr/impl/src/lib.rs

```diff
@@ -34,6 +34,7 @@
 #![feature(result_flattening)]
 
 mod aggregate;
+mod external;
```

Comment:
Should we put this into `scalar` module?

Reply:
LGTM